data/json/furniture.json

```diff
@@ -2769,5 +2769,34 @@
                 { "item": "rock", "count": [18, 30] }
             ]
         }
+    },
+    {
+        "type" : "furniture",
+        "id" : "f_makeshift_nest",
+        "name": "makefhist nest",
+        "symbol": "O",
+        "color": "brown",
+        "move_cost_mod": 1,
+        "required_str": 5,
+        "deconstruct": {
+            "items": [
+                { "item": "2x4", "count": 10 },
+                { "item": "rag", "count": [30, 33] },
```

Comment:
Deconstruction this gives *much* more rags and 2x4s than constructing it needs. Same with rags and bashing (construction requires 10 rags, bashing give 15-20).